src/collective/glossary/browser/views.py

```diff
@@ -1,12 +1,24 @@
-# -*- coding: utf-8 -*-
-
 from collections import defaultdict
 from collective.glossary.interfaces import IGlossarySettings
+from functools import cached_property
 from plone import api
 from plone.i18n.normalizer.base import baseNormalize
 from Products.Five.browser import BrowserView
 
 import json
+import logging
+import os
+
+
+logger = logging.getLogger(__name__)
+
+# Maximum amount of entries before we show the A-Z toolbar:
+env_name = "COLLECTIVE_GLOSSARY_MAXIMUM_WITHOUT_AZ_TOOLBAR"
+try:
+    MAXIMUM_WITHOUT_AZ_TOOLBAR = int(os.environ.get(env_name))
+    logger.info("Using %d as maximum without AZ toolbar.", MAXIMUM_WITHOUT_AZ_TOOLBAR)
+except Exception:
+    MAXIMUM_WITHOUT_AZ_TOOLBAR = 30
```

Comment:
You may want to use collective.glossary controlpanel, instead of an environment variable, to let the site admin to choose from how many entries on an a-z toolbar is shown.

Reply:
Ah right, good point.  Done, including an upgrade step.  Can you check again please?